superduperdb/misc/datatype_helper.py

```diff
@@ -0,0 +1,22 @@
+from superduperdb import logging
+from superduperdb.components.datatype import DataType
+
+
+def convert_model_datatype(datatype, ibis=False):
```

Comment:
If we have many databackend implementations, this logic won't scale.

Reply:
A better approach is that each data backend should have its own corresponding conversion method.

Superduperdb will have a schema that outputs a uniform format. Different backends need to adapt to this schema accordingly.

For example, if Superduperdb only outputs vector types, different data backends should adjust themselves after receiving it.